### DISCUSSION TITLE
Add HeadlessDocument menu not showing

### DISCUSSION THREAD

**Discussion by oivarc** [pimcore/pimcore#17406] [Q&A / Support]
Hello,

We have upgraded to Pimcore 11x and added the Headless Documents bundle. We can create the headless templates and brick, but in the context menu we do not get the option to Add Headless document. I see that also on the Pimcore Live demo the menu to Add Headless document is not there. 

Debugging the javascript startup.js in the bundle, in the initialize function it is using the pimcore.events.preMenuBuild that is not initializing the  headless_documents_store. I have tried to use the pimcore.events.pimcoreReady listener (commenting the line that is using the pimcore.events.preMenuBuild) and it is working fine.

Is it an issue with the bundle or with our installation ?

**Comment by fashxp** [comment]
this is a bug, will be fixed in next bugfix release.

**Comment by martineiber** [comment]
Bugfix release 2.1.4 was just released.
